server/sync.go

```diff
@@ -186,9 +186,29 @@ func (server *Server) syncEvent(resource *schema.Resource) error {
 	}
 	defer tx.Close()
 	eventType := resource.Get("type").(string)
-	path := resource.Get("path").(string)
-	path = configPrefix + path
+	resourcePath := resource.Get("path").(string)
 	body := resource.Get("body").(string)
+
+	var curSchema = schema.GetSchemaByPath(resourcePath)
+
+	path := resourcePath
```

Comment:
It seems it'll be good to extract this part (`path` generation) into a function.